homeassistant/components/zwave_js/manifest.json

```
@@ -8,8 +8,8 @@
   "dependencies": ["usb", "http", "websocket_api"],
   "iot_class": "local_push",
   "usb": [
-   {"vid":"0658","pid":"0200"},
-   {"vid":"10C4","pid":"8A2A"},
-   {"vid":"10C4","pid":"EA60"}
+   {"vid":"0658","pid":"0200","known_devices":["Aeotec Z-Stick Gen5+", "Z-WaveMe UZB"]},
+   {"vid":"10C4","pid":"8A2A","known_devices":["Nortek HUSBZB-1"]},
+   {"vid":"10C4","pid":"EA60","known_devices":["Zooz ZST10"]}
```

The Zooz listed here is actually the "Zooz ZST10 700" (700 series). The plain "Zooz ZST10" is the 500 series controller and _should_ have vid/pid `0658:0200`, same as the Aeotec (there are many products that share this).

Similarly, `10C4:EA60` is shared by all of the currently known 700 series USB sticks, including Silicon Labs SLUSB001A (UZB-7), Aeotec Z-Stick 7, Zooz ZST10 700, and I would presume others. It actually corresponds to the USB-to-UART SoC, which is why the ZHA cc2652rb stick matches.

Would it make sense to just list this generically to cover all current and future variations of the same product, or do you want to list all of those others? I'm not sure what the plans are for this information, but listing only the Zooz might be confusing.

```suggestion
   {"vid":"10C4","pid":"EA60","known_devices":["Z-Wave 700 Series Controller"]}
```

---

I think we should just list them all in that case...

---

In that case, I would suggest:

```suggestion
   {"vid":"10C4","pid":"EA60","known_devices":["Aeotec Z-Stick 7", "Silicon Labs UZB-7", "Zooz ZST10 700"]}
```

There are a [few other](https://products.z-wavealliance.org/Search/DoAdvancedSearch?productName=&productIdentifier=&productDescription=usb&category=11&brand=-1&regionId=-1&zwavePlus2Only=on&supportsS2=on&supportsSmartStart=on&zwaveNewTech=on&order=) certified USB controllers, but I can't vouch for them, especially since they are listed with a different hardware platform. Probably better to add as needed.

---

> I think we should just list them all in that case... and not generalize, as that makes things less clear.

I agree, these names are for humans so lets make it easy for them to match to a device